ui/components/Modals/Information/InfoModal.js

```diff
@@ -432,7 +432,7 @@ const InfoModal_ = React.memo((props) => {
               onClick={handlePublishController}
               className={classes.copyButton}
               disabled={
-                isAdmin
+                isAdmin && !isPublished
```

Comment:
@shubham251972 why is "isAdmin" being used an not keys?

Reply:
Ohh ho! no there is no use, we can now remove it as we are using keys, I'll remove it

Reply:
No, there is no use of it, I'll remove it

Reply:
Keys are already being use, it's just we might have forget to remove this isAdmin

Reply:
@shubham251972 changes made?

Reply:
@shubham251972 changes made?